web/packages/app/package.json

```diff
@@ -47,22 +47,22 @@
     "stylelint-config-standard": "^28.0.0",
     "ts-node": "^10.8.1",
     "typescript": "^4.7.3",
+    "vm-browserify": "^1.1.2",
     "webpack": "^5.94.0",
     "webpack-cli": "^4.9.2",
-    "webpack-dev-server": "^4.9.1",
-    "vm-browserify": "^1.1.2"
+    "webpack-dev-server": "^4.9.1"
   },
   "dependencies": {
+    "@codingame/monaco-vscode-files-service-override": "~11.1.1",
+    "@codingame/monaco-vscode-keybindings-service-override": "~11.1.1",
+    "@codingame/monaco-vscode-textmate-service-override": "~11.1.1",
+    "@codingame/monaco-vscode-theme-defaults-default-extension": "~11.1.1",
     "debounce": "^2.2.0",
     "json-rpc-2.0": "^1.3.0",
     "monaco-editor-core": "^0.52.0",
     "monaco-editor-wrapper": "~6.0.0-next.9",
-    "vscode-languageserver-protocol": "^3.17.5",
-    "vscode-languageclient": "^9.0.1",
     "vscode": "npm:@codingame/monaco-vscode-api@~11.1.1",
-    "@codingame/monaco-vscode-keybindings-service-override": "~11.1.1",
-    "@codingame/monaco-vscode-theme-defaults-default-extension": "~11.1.1",
-    "@codingame/monaco-vscode-textmate-service-override": "~11.1.1",
-    "@codingame/monaco-vscode-files-service-override": "~11.1.1"
+    "vscode-languageclient": "^9.0.1",
+    "vscode-languageserver-protocol": "^3.17.5"
   }
 }
```

Comment:
These look harmless, but I was surprised to see this file in the diff.

Reply:
Ah, I did `npm audit fix` or something similar to silence some warnings.